src/qtt/utilities/tools.py

```diff
@@ -850,10 +850,10 @@ def figure_ppt_callback(event):
     import win32com
     import win32com.client
 
-
     def addPPTslide(title=None, fig=None, txt=None, notes=None, figsize=None,
                     subtitle=None, maintext=None, show=False, verbose=1,
-                    activate_slide=True, ppLayout=None, extranotes=None, background_color=None):
+                    activate_slide=True, ppLayout=None, extranotes=None, background_color=None,
+                    maximum_notes_size: int = 10e3):
```

Comment:
When you test it, it is implicitly 'converted' to 10000. Not what you want.

Reply:
10000 is indeed quite some text, but i know that there are notes added of this size (because the snapshot is included) so I would not want to restrict more be default. 

A user can off course change the default value